pySDC/core/sweeper.py

```diff
@@ -251,6 +247,8 @@ def level(self, L):
         Args:
             L (pySDC.Level.level): current level
         """
+        from pySDC.core.level import Level
```

Comment:
Why is this here and not on top?

Reply:
Because it creates a circular dependency that produce an error on import